utils/validations/test_validation.go

```diff
@@ -0,0 +1,164 @@
+package validations
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jfrog/jfrog-cli-security/utils"
+)
+
+type ValidationParams struct {
+	// The actual content to verify.
+	Actual interface{}
+	// If provided, the test will check if the content matches the expected results.
+	Expected interface{}
```

Comment:
what is the difference between the 'Expected' field and the values below to which you expect?